src/main/resources/assignment/birds/birds-view.fxml

```diff
@@ -40,12 +40,12 @@
                                 <Insets right="10.0" />
                             </HBox.margin>
                         </Button>
-                        <Button fx:id="next" minWidth="-Infinity" mnemonicParsing="false" onAction="#next" prefHeight="28.0" prefWidth="90.0" style="-fx-background-color: lightgreen;" text="Next">
+                        <Button fx:id="previous" minWidth="-Infinity" mnemonicParsing="false" onAction="#previous" prefHeight="27.0" prefWidth="90.0" style="-fx-background-color: lightgreen;" text="Previous">
```

Comment:
this was driving me nuts when testing, hope this is okay (also why is prefHeight different between these two? T_T)